k8s/upgrades/0.8.2-0.9.0/cstor/cstor-pool-update-082-090.yaml

```diff
@@ -415,7 +497,7 @@ spec:
   post: |-
     {{- jsonpath .JsonResult "{.spec.template.spec.containers[?(@.name=='cstor-pool')].image}" | trim | saveAs "postCheckDeploymentImagePatch.cstorpoolimage" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.spec.template.spec.containers[?(@.name=='cstor-pool-mgmt')].image}" | trim | saveAs "postCheckDeploymentImagePatch.cstorpoolmgmtimage" .TaskResult | noop -}}
-    {{- jsonpath .JsonResult "{.spec.template.spec.containers[?(@.name=='maya-exporter')].image}" | trim | saveAs "postCheckDeploymentImagePatch.mayaexporterimage" .TaskResult | noop -}}
+    {{- jsonpath .JsonResuluidate.spec.containers[?(@.name=='maya-exporter')].image}" | trim | saveAs "postCheckDeploymentImagePatch.mayaexporterimage" .TaskResult | noop -}}
```

Comment:
this seems to be error

Reply:
While removing debug logs it got edited will fix it